src/main/resources/standardTestCases/fofn_caching.test

```diff
@@ -0,0 +1,13 @@
+name: fofn_caching
+testFormat: runtwiceexpectingcallcaching
+backends: [Jes]
```

Comment:
We might want to create another backend for this test with the "don't copy" config turned on and leave the current behavior on the current Jes Backend ?